aws/s3/pkg/commands/delete_object.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"log"
+)
+
+type DeleteObjectParams struct {
+	Bucket string
+	Key    string
+}
+
+func DeleteObject(ctx context.Context, client *s3.Client, params DeleteObjectParams) error {
+	if params.Bucket == "" {
+		return fmt.Errorf("bucket name is required")
+	}
+	if params.Key == "" {
+		return fmt.Errorf("key is required")
+	}
+
+	input := &s3.DeleteObjectInput{
+		Bucket: aws.String(params.Bucket),
+		Key:    aws.String(params.Key),
+	}
+
+	_, err := client.DeleteObject(ctx, input)
+
```

Comment:
nit: check the error as soon as you can. There are a few other spots.
```suggestion
```

Reply:
think i got these all resolved